pwiz_tools/Skyline/TestFunctional/DdaSearchTest.cs

```diff
@@ -109,6 +109,13 @@ public void TestDdaSearch()
         [TestMethod]
         public void TestDdaSearchMsgfPlus()
         {
+            if (Helpers.RunningResharperAnalysis)
+            {
+                // This test just takes much too long under code coverage analysis
+                Console.Write(MSG_SKIPPING_SLOW_RESHARPER_ANALYSIS_TEST);
```

Comment:
We really have nothing better than writing to "Console"? I feel like we should have something more formal, like a method on TestContext, to distinguish between code that we really want to keep as important testing output and code that accidentally got left behind during debugging, which we see plenty of.

Reply:
Ultimately we do want Console output, but I take your point about putting that in just one place where it's clear that it's intentional.